packages/atomic/src/components/atomic-layout/atomic-search-layout.tsx

```diff
@@ -18,7 +18,7 @@ export class AtomicSearchLayout {
    */
   @Prop({reflect: true}) public mobileBreakpoint = '1024px';
 
-  public connectedCallback() {
+  public componentDidLoad() {
```

Comment:
On connectedCallback, the props of the components in Stencil's JSX are not readable even when reflected